### PR TITLE
Update quickstart_firstlogin.md

### PR DESCRIPTION
**Pull request by michaelabela**

### CODE REVIEW

File: docs/researchers/codex/quickstart_firstlogin.md [hunk 1]
```diff
@@ -9,7 +9,11 @@ This Quickstart topic covers some basic tasks and concepts when first logging in
 
 ## Account Registration
 
-TBD
+To get set up with an account, use [this link](https://qa.dyn.wildme.io/request) to access Codex’s invitation request page. Alternatively, you may click “Request an invitation” on the Login page or at the very bottom of the Welcome page under the Account section of the footer.
```
Comment (Atticus29) [inline docs/researchers/codex/quickstart_firstlogin.md]: This URL takes a user to our QA server. Do we really want that in the quickstart docs?

I think we should be a little more agnostic about which platform the reader is using.

Perhaps something more general like, "navigate to your codex instance's URL, and add /request to the end of the URL")?

**Review by Atticus29 — CHANGES_REQUESTED** [review]

File: docs/researchers/codex/quickstart_firstlogin.md (hunk 1 above)
Comment (michaelabela) [reply docs/researchers/codex/quickstart_firstlogin.md]: good catch, yes that definitely makes more sense

**Review by TanyaStere42 — CHANGES_REQUESTED** [review]
ln 12: these are general documents, not for our QA server specifically, which makes that first sentence misleading (not your fault, directions were unclear). Detail how to navigate from a site home page to the log-in page.
rest looks great

**Review by Atticus29 — APPROVED** [review]
LGTM.

**Review by TanyaStere42 — APPROVED** [review]

**Pull request merged**